README

```diff
@@ -23,26 +23,49 @@ Running `htop` requires `ncurses` libraries (typically named libncursesw*).
 For more information and details on how to contribute to `htop` visit [htop.dev](https://htop.dev).
 
 ## Build instructions
-
-This program is distributed as a standard GNU autotools-based package.
-
-Compiling `htop` requires the header files for `ncurses` (libncursesw*-dev). Install these and other required packages for C development from your package manager.
-
-Then, when compiling from a [release tarball](https://github.com/htop-dev/htop/releases), run:
-
+### Prerequisite
+List of build-time dependencies:
+*  `build-essential` standard GNU autotools-based
+*  `autoconf`
+*  `autotools`
+*  `ncurses` (libncursesw*-dev)
+
+**Note about `ncurses`:**
+> htop requires ncurses 6.0, the appropriate package is often enough still called libncurses5 (+-dev for headers). Also ncurses usually comes in two flavours:
+>* With Unicode support.
+>* Without Unicode support.
+>
+> This is also something that is usually reflected in the package name (the additional w).
+
+List of additional build-time dependencies (based on feature flags):
+*  `sensors`
+*  `hwloc`
+*  `libcap`
+
+Compiling `htop` requires the header files for `ncurses` . Install these and other required packages for C development from your package manager.
+
+**Debian/ubuntu**
 ~~~ shell
-./configure && make
+sudo apt install libncursesw5-dev autotools-dev autoconf
 ~~~
 
-Alternatively, for compiling sources downloaded from the Git repository (`git clone` or downloads from [Github releases](https://github.com/htop-dev/htop/releases/)),
-install the header files for `ncurses` (libncursesw*-dev) and other required development packages from your distribution's package manager. Then run:
+List of additional build-time dependencies (based on feature flags):
+*  `sensors`
+*  `hwloc`
+*  `ncurses`
```

Comment:
`ncurses` is always required. The only difference is whether the non-Unicode or the Unicode/wide-char version of it is required.